app/stories/Alert.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Alert, Button, Icon, Icons } from "@phoenix/components";
+
+const meta: Meta<typeof Alert> = {
+  title: "Components/Alert",
```

Comment:
This is putting the story into its own category.

Also duplicated with the story in src/stories

<img width="293" alt="image" src="https://github.com/user-attachments/assets/a1c013b1-b0b1-4f66-8553-41fb04cd0c34" />